File: gherkin/java/src/test/java/io/cucumber/gherkin/TokensGenerator.java
package io.cucumber.gherkin;

public class TokensGenerator
{
	public static String generateTokens(String featureContent
		, Parser.ITokenMatcher tokenMatcher)
	{
		TokenFormatterBuilder tokenFormatterBuilder = new TokenFormatterBuilder();
		Parser<String> parser = new Parser<>(tokenFormatterBuilder);
		return parser.parse(featureContent, tokenMatcher);
	}
}

File: gherkin/dart/lib/src/tokens/TokensGenerator.dart
import 'package:gherkin/helpers.dart';
import 'package:gherkin/language.dart';
import 'package:gherkin/parser.dart';
import 'TokenFormatterBuilder.dart';
import 'object.dart';

class TokensGenerator
{
  static String generateTokens(String featureFilePath, ITokenMatcher tokenMatcher) {
    var tokenFormatterBuilder = TokenFormatterBuilder();
    var parser = Parser<object>(tokenFormatterBuilder);
    var tokenScanner = FileTokenScanner.fromPath(featureFilePath);
    parser.parse(tokenScanner, tokenMatcher);
    var tokensText = tokenFormatterBuilder.getTokensText();
    return LineEndingHelper.normalizeLineEndings(tokensText);
  }
}
